### DISCUSSION TITLE
Adding AccountingDB as a data source in WLCG Grafana organisation

### DISCUSSION THREAD

**Discussion by EwoudK** [DIRACGrid/DIRAC#6981] [Q&A]
Hi
I'm not sure what the proper channel is to ask this question so feel free to redirect me.

I'm working in the WLCG Grafana organistion on a Grafana dashboard that should replace the currently static WLCG RRB report. For this, I would need (to start) the numbers of cores used per day per experiment. I already identified the necessary query to extract this data for LHCb from the AccountingDB but now I would also need to add the AccountingDB as a data source to the WLCG Grafana organisation so I can execute this query from Grafana. Would it be possible to create database credentials with (limited) access that I can use for this?

**Comment by fstagni** [comment]
I close this one, as discussed elsewhere.
